go.mod

```diff
@@ -23,14 +23,13 @@ require (
 	github.com/qeesung/image2ascii v1.0.1
 	github.com/rs/zerolog v1.33.0
 	github.com/saferwall/pe v1.5.4
-	github.com/santhosh-tekuri/jsonschema/v6 v6.0.1
-	github.com/santhosh-tekuri/jsonschema/v6 v6.0.1
+	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
```

Comment:
why the need to downgrade?

Reply:
Comes from here: https://github.com/kairos-io/kairos-sdk/blob/main/schema/root_schema.go#L7 after updating the go mod